dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -273,7 +273,10 @@ def dpnp_abs(x, out=None, order="K"):
     out_usm = None if out is None else dpnp.get_usm_ndarray(out)
 
     res_usm = abs_func(x1_usm, out=out_usm, order=order)
-    return dpnp_array._create_from_usm_ndarray(res_usm)
+    if out is None:
+        return dpnp_array._create_from_usm_ndarray(res_usm)
+    else:
+        return out
```

Comment:
probably it makes sense to reduce code duplication with an internal function, like:
```python
def _get_result(res_usm, out=None):
    if out is None:
        return dpnp_array._create_from_usm_ndarray(res_usm)
    else:
        return out
```
and so:
```suggestion
    return _get_result(res_usm, out=out)
```